source/ClientApp/src/app/vendors/Models/fake-vendor.model.ts

```diff
@@ -6,7 +6,6 @@ export class FakeVendor implements Vendor {
     id = 1001;
     userName = 'vendor@email.com';
     name = 'Vendor Name';
-    type = 'VENDOR';
```

Comment:
Might want to put this into some kind of constants file in the future for standardization